spring-cloud-kubernetes-integration-tests/run.sh

```diff
@@ -87,7 +87,7 @@ main() {
     cd $CURRENT_DIR
 
     #TODO what happens if cluster is already there????
-    "${KIND}" create cluster --config=kind-config.yaml --loglevel=debug
+    "${KIND}" create cluster --config=kind-config.yaml -v 2147483647
```

Comment:
What is the `-v` param?

Reply:
`--loglevel=debug` is deprecated. `-v 2147483647` is like saying "max output possible", sort of like trace